packages/react-native/src/primitives/TextField/__tests__/TextField.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import TextField from '../TextField';
+
+const placeHolderText = 'Placeholder';
+const labelText = 'Label';
+const testID = 'textField';
+const defaultProps = {
+  testID: testID,
+  label: labelText,
+  placeholder: placeHolderText,
+};
+
+const onChangeText = jest.fn();
+
+describe('TextField', () => {
+  it('renders as expected', async () => {
+    const { toJSON, findAllByRole, getByPlaceholderText, getByText } = render(
+      <TextField {...defaultProps} />
+    );
+    expect(toJSON()).toMatchSnapshot();
+
+    expect(await findAllByRole('text')).toHaveLength(1);
+    const textInput = getByPlaceholderText(placeHolderText);
+    expect(textInput.props.editable).toBeTruthy();
+    expect(textInput.props.secureTextEntry).toBeFalsy();
+    expect(textInput.parent?.props.accessible).toBeTruthy();
```

Comment:
Curious abt the optional chaining here? Is it just a TS requirement?

Reply:
yeah ts req, parent may be null.